### ISSUE TITLE
[1.10.0rc3] No such file or directory: 'build\\lib\\src/octoprint\\_static_version.py' when installing on Windows

### ISSUE THREAD

**Issue by MaleNurse** [OctoPrint/OctoPrint#4987]
### The problem

When installing clean or updating OctoPrint on a Windows machine, the install errors out on "No such file or directory: 'build\\lib\\src/octoprint\\_static_version.py'"

I've tried this both by trying to update a v1.9.3 install on Windows Server 2022 running Python 3.11 as well as installing clean on a Win10 box running Python 3.12. This also started w/ the first rc for 1.10.0.

I've included the output of the install in the files section below.

### Did the issue persist even in safe mode?

Yes, it did persist

### Version of OctoPrint

1.10.0rc3

### Operating system running OctoPrint

Windows 10 and Windows Server 2022

### Checklist of files to include below

- [X] Systeminfo Bundle (always include!)
- [ ] Contents of the JavaScript browser console (always include in cases of issues with the user interface)
- [ ] Screenshots and/or videos showing the problem (always include in case of issues with the user interface)
- [ ] GCODE file with which to reproduce (always include in case of issues with GCODE analysis or printing behaviour)

### Additional information & file uploads

[octoprint-systeminfo-20240407171831.zip](https://github.com/OctoPrint/OctoPrint/files/14899595/octoprint-systeminfo-20240407171831.zip)
[install.log](https://github.com/OctoPrint/OctoPrint/files/14899596/install.log)

**Comment by foosel** [comment]
Confirmed when installing from a source tarball (which `get.octoprint.org` links to). 

Fix ready for 1.10.0rc4.

What I don't understand is why I see over 500 instances running 1.10.0rc* under Windows... The only explanation that I have is that they installed from PyPi which likely defaulted to the ready made wheel which doesn't have this issue:

```
(testenv) PS C:\Users\gina> pip install OctoPrint==1.10.0rc3
Collecting OctoPrint==1.10.0rc3
  Downloading OctoPrint-1.10.0rc3-py2.py3-none-any.whl.metadata (12 kB)
Collecting OctoPrint-FileCheck>=2021.2.23 (from OctoPrint==1.10.0rc3)
  Using cached OctoPrint_FileCheck-2024.3.27-py2.py3-none-any.whl.metadata (1.7 kB)
Collecting OctoPrint-FirmwareCheck>=2021.10.11 (from OctoPrint==1.10.0rc3)
  Using cached OctoPrint_FirmwareCheck-2021.10.11-py2.py3-none-any.whl.metadata (3.9 kB)
Collecting OctoPrint-PiSupport>=2023.10.10 (from OctoPrint==1.10.0rc3)
  Using cached OctoPrint_PiSupport-2023.10.10-py2.py3-none-any.whl.metadata (2.9 kB)
Collecting argon2-cffi>=23.1.0 (from OctoPrint==1.10.0rc3)
  Using cached argon2_cffi-23.1.0-py3-none-any.whl.metadata (5.2 kB)
Collecting Babel<2.13,>=2.12.1 (from OctoPrint==1.10.0rc3)
  Using cached Babel-2.12.1-py3-none-any.whl.metadata (1.3 kB)
Collecting cachelib<0.11,>=0.10.2 (from OctoPrint==1.10.0rc3)
  Using cached cachelib-0.10.2-py3-none-any.whl.metadata (2.0 kB)
Collecting Click<9,>=8.1.7 (from OctoPrint==1.10.0rc3)
  Using cached click-8.1.7-py3-none-any.whl.metadata (3.0 kB)
Collecting colorlog<7,>=6.7.0 (from OctoPrint==1.10.0rc3)
  Using cached colorlog-6.8.2-py3-none-any.whl.metadata (10 kB)
Collecting emoji<3,>=2.10.1 (from OctoPrint==1.10.0rc3)
  Using cached emoji-2.11.0-py2.py3-none-any.whl.metadata (5.3 kB)
Collecting feedparser<7,>=6.0.11 (from OctoPrint==1.10.0rc3)
  Using cached feedparser-6.0.11-py3-none-any.whl.metadata (2.4 kB)
Collecting filetype<2,>=1.2.0 (from OctoPrint==1.10.0rc3)
  Using cached filetype-1.2.0-py2.py3-none-any.whl.metadata (6.5 kB)
Collecting Flask-Assets<3,>=2.1.0 (from OctoPrint==1.10.0rc3)
  Using cached Flask_Assets-2.1.0-py3-none-any.whl.metadata (931 bytes)
Collecting Flask-Babel<4,>=3.1.0 (from OctoPrint==1.10.0rc3)
  Using cached flask_babel-3.1.0-py3-none-any.whl.metadata (2.2 kB)
Collecting Flask-Login<0.7,>=0.6.3 (from OctoPrint==1.10.0rc3)
  Using cached Flask_Login-0.6.3-py3-none-any.whl.metadata (5.8 kB)
Collecting Flask-Limiter<4,>=3.5.0 (from OctoPrint==1.10.0rc3)
  Using cached Flask_Limiter-3.5.1-py3-none-any.whl.metadata (6.1 kB)
Collecting flask<2.3,>=2.2.3 (from OctoPrint==1.10.0rc3)
  Using cached Flask-2.2.5-py3-none-any.whl.metadata (3.9 kB)
Collecting frozendict<3,>=2.4.0 (from OctoPrint==1.10.0rc3)
  Using cached frozendict-2.4.1-cp312-cp312-win_amd64.whl
Collecting future<1,>=0.18.3 (from OctoPrint==1.10.0rc3)
  Using cached future-0.18.3-py3-none-any.whl
Collecting markdown<3.5,>=3.4.4 (from OctoPrint==1.10.0rc3)
  Using cached Markdown-3.4.4-py3-none-any.whl.metadata (6.9 kB)
Collecting netaddr<0.9,>=0.8 (from OctoPrint==1.10.0rc3)
  Using cached netaddr-0.8.0-py2.py3-none-any.whl.metadata (4.9 kB)
Collecting netifaces2<0.1,>=0.0.21 (from OctoPrint==1.10.0rc3)
  Using cached netifaces2-0.0.22-cp37-abi3-win_amd64.whl.metadata (5.3 kB)
Collecting passlib<2,>=1.7.4 (from OctoPrint==1.10.0rc3)
  Using cached passlib-1.7.4-py2.py3-none-any.whl.metadata (1.7 kB)
Collecting pathvalidate<3,>=2.5.2 (from OctoPrint==1.10.0rc3)
  Using cached pathvalidate-2.5.2-py3-none-any.whl.metadata (11 kB)
Collecting pkginfo<2,>=1.9.6 (from OctoPrint==1.10.0rc3)
  Using cached pkginfo-1.10.0-py3-none-any.whl.metadata (11 kB)
Collecting psutil<6,>=5.9.8 (from OctoPrint==1.10.0rc3)
  Using cached psutil-5.9.8-cp37-abi3-win_amd64.whl.metadata (22 kB)
Collecting pydantic==1.10.12 (from OctoPrint==1.10.0rc3)
  Using cached pydantic-1.10.12-py3-none-any.whl.metadata (149 kB)
Collecting pylru<2,>=1.2.1 (from OctoPrint==1.10.0rc3)
  Using cached pylru-1.2.1-py3-none-any.whl.metadata (13 kB)
Collecting pyserial<4,>=3.5 (from OctoPrint==1.10.0rc3)
  Using cached pyserial-3.5-py2.py3-none-any.whl.metadata (1.6 kB)
Collecting pytz (from OctoPrint==1.10.0rc3)
  Using cached pytz-2024.1-py2.py3-none-any.whl.metadata (22 kB)
Collecting PyYAML<7,>=6.0.1 (from OctoPrint==1.10.0rc3)
  Using cached PyYAML-6.0.1-cp312-cp312-win_amd64.whl.metadata (2.1 kB)
Collecting requests<3,>=2.31.0 (from OctoPrint==1.10.0rc3)
  Using cached requests-2.31.0-py3-none-any.whl.metadata (4.6 kB)
Collecting sarge==0.1.7.post1 (from OctoPrint==1.10.0rc3)
  Using cached sarge-0.1.7.post1-py2.py3-none-any.whl.metadata (1.2 kB)
Collecting semantic-version<3,>=2.10.0 (from OctoPrint==1.10.0rc3)
  Using cached semantic_version-2.10.0-py2.py3-none-any.whl.metadata (9.7 kB)
Collecting sentry-sdk<2,>=1.40.0 (from OctoPrint==1.10.0rc3)
  Using cached sentry_sdk-1.44.1-py2.py3-none-any.whl.metadata (9.9 kB)
Collecting setuptools (from OctoPrint==1.10.0rc3)
  Using cached setuptools-69.2.0-py3-none-any.whl.metadata (6.3 kB)
Collecting tornado<6.3,>=6.2 (from OctoPrint==1.10.0rc3)
  Using cached tornado-6.2-cp37-abi3-win_amd64.whl.metadata (2.5 kB)
Collecting watchdog<3,>=2.3.1 (from OctoPrint==1.10.0rc3)
  Using cached watchdog-2.3.1-py3-none-win_amd64.whl.metadata (36 kB)
Collecting websocket-client==1.6.1 (from OctoPrint==1.10.0rc3)
  Using cached websocket_client-1.6.1-py3-none-any.whl.metadata (7.6 kB)
Collecting werkzeug<2.3,>=2.2.3 (from OctoPrint==1.10.0rc3)
  Using cached Werkzeug-2.2.3-py3-none-any.whl.metadata (4.4 kB)
Collecting wrapt<1.16,>=1.15 (from OctoPrint==1.10.0rc3)
  Using cached wrapt-1.15.0-py3-none-any.whl.metadata (6.7 kB)
Collecting zeroconf~=0.127 (from OctoPrint==1.10.0rc3)
  Using cached zeroconf-0.132.0-cp312-cp312-win_amd64.whl
Collecting zipstream-ng<2.0.0,>=1.7.1 (from OctoPrint==1.10.0rc3)
  Using cached zipstream_ng-1.7.1-py3-none-any.whl.metadata (10 kB)
Collecting blinker<1.7.0,>=1.6.3 (from OctoPrint==1.10.0rc3)
  Using cached blinker-1.6.3-py3-none-any.whl.metadata (1.9 kB)
Collecting class-doc<0.3,>=0.2.6 (from OctoPrint==1.10.0rc3)
  Using cached class_doc-0.2.6-py3-none-any.whl.metadata (2.9 kB)
Collecting regex (from OctoPrint==1.10.0rc3)
  Using cached regex-2023.12.25-cp312-cp312-win_amd64.whl.metadata (41 kB)
Collecting unidecode (from OctoPrint==1.10.0rc3)
  Using cached Unidecode-1.3.8-py3-none-any.whl.metadata (13 kB)
Collecting wheel (from OctoPrint==1.10.0rc3)
  Using cached wheel-0.43.0-py3-none-any.whl.metadata (2.2 kB)
Collecting typing-extensions>=4.2.0 (from pydantic==1.10.12->OctoPrint==1.10.0rc3)
  Using cached typing_extensions-4.11.0-py3-none-any.whl.metadata (3.0 kB)
Collecting argon2-cffi-bindings (from argon2-cffi>=23.1.0->OctoPrint==1.10.0rc3)
  Using cached argon2_cffi_bindings-21.2.0-cp36-abi3-win_amd64.whl.metadata (6.7 kB)
Collecting more-itertools>=5.0.0 (from class-doc<0.3,>=0.2.6->OctoPrint==1.10.0rc3)
  Using cached more_itertools-10.2.0-py3-none-any.whl.metadata (34 kB)
Collecting colorama (from Click<9,>=8.1.7->OctoPrint==1.10.0rc3)
  Using cached colorama-0.4.6-py2.py3-none-any.whl.metadata (17 kB)
Collecting sgmllib3k (from feedparser<7,>=6.0.11->OctoPrint==1.10.0rc3)
  Using cached sgmllib3k-1.0.0-py3-none-any.whl
Collecting Jinja2>=3.0 (from flask<2.3,>=2.2.3->OctoPrint==1.10.0rc3)
  Using cached Jinja2-3.1.3-py3-none-any.whl.metadata (3.3 kB)
Collecting itsdangerous>=2.0 (from flask<2.3,>=2.2.3->OctoPrint==1.10.0rc3)
  Using cached itsdangerous-2.1.2-py3-none-any.whl.metadata (2.9 kB)
Collecting webassets>=2.0 (from Flask-Assets<3,>=2.1.0->OctoPrint==1.10.0rc3)
  Using cached webassets-2.0-py3-none-any.whl.metadata (889 bytes)
Collecting limits>=2.8 (from Flask-Limiter<4,>=3.5.0->OctoPrint==1.10.0rc3)
  Using cached limits-3.10.1-py3-none-any.whl.metadata (7.1 kB)
Collecting ordered-set<5,>4 (from Flask-Limiter<4,>=3.5.0->OctoPrint==1.10.0rc3)
  Using cached ordered_set-4.1.0-py3-none-any.whl.metadata (5.3 kB)
Collecting rich<14,>=12 (from Flask-Limiter<4,>=3.5.0->OctoPrint==1.10.0rc3)
  Using cached rich-13.7.1-py3-none-any.whl.metadata (18 kB)
Collecting charset-normalizer<4,>=2 (from requests<3,>=2.31.0->OctoPrint==1.10.0rc3)
  Using cached charset_normalizer-3.3.2-cp312-cp312-win_amd64.whl.metadata (34 kB)
Collecting idna<4,>=2.5 (from requests<3,>=2.31.0->OctoPrint==1.10.0rc3)
  Using cached idna-3.6-py3-none-any.whl.metadata (9.9 kB)
Collecting urllib3<3,>=1.21.1 (from requests<3,>=2.31.0->OctoPrint==1.10.0rc3)
  Using cached urllib3-2.2.1-py3-none-any.whl.metadata (6.4 kB)
Collecting certifi>=2017.4.17 (from requests<3,>=2.31.0->OctoPrint==1.10.0rc3)
  Using cached certifi-2024.2.2-py3-none-any.whl.metadata (2.2 kB)
Collecting MarkupSafe>=2.1.1 (from werkzeug<2.3,>=2.2.3->OctoPrint==1.10.0rc3)
  Using cached MarkupSafe-2.1.5-cp312-cp312-win_amd64.whl.metadata (3.1 kB)
Collecting ifaddr>=0.1.7 (from zeroconf~=0.127->OctoPrint==1.10.0rc3)
  Using cached ifaddr-0.2.0-py3-none-any.whl.metadata (4.9 kB)
Collecting deprecated>=1.2 (from limits>=2.8->Flask-Limiter<4,>=3.5.0->OctoPrint==1.10.0rc3)
  Using cached Deprecated-1.2.14-py2.py3-none-any.whl.metadata (5.4 kB)
Collecting importlib-resources>=1.3 (from limits>=2.8->Flask-Limiter<4,>=3.5.0->OctoPrint==1.10.0rc3)
  Using cached importlib_resources-6.4.0-py3-none-any.whl.metadata (3.9 kB)
Collecting packaging<25,>=21 (from limits>=2.8->Flask-Limiter<4,>=3.5.0->OctoPrint==1.10.0rc3)
  Using cached packaging-24.0-py3-none-any.whl.metadata (3.2 kB)
Collecting markdown-it-py>=2.2.0 (from rich<14,>=12->Flask-Limiter<4,>=3.5.0->OctoPrint==1.10.0rc3)
  Using cached markdown_it_py-3.0.0-py3-none-any.whl.metadata (6.9 kB)
Collecting pygments<3.0.0,>=2.13.0 (from rich<14,>=12->Flask-Limiter<4,>=3.5.0->OctoPrint==1.10.0rc3)
  Using cached pygments-2.17.2-py3-none-any.whl.metadata (2.6 kB)
Collecting cffi>=1.0.1 (from argon2-cffi-bindings->argon2-cffi>=23.1.0->OctoPrint==1.10.0rc3)
  Using cached cffi-1.16.0-cp312-cp312-win_amd64.whl.metadata (1.5 kB)
Collecting pycparser (from cffi>=1.0.1->argon2-cffi-bindings->argon2-cffi>=23.1.0->OctoPrint==1.10.0rc3)
  Using cached pycparser-2.22-py3-none-any.whl.metadata (943 bytes)
Collecting mdurl~=0.1 (from markdown-it-py>=2.2.0->rich<14,>=12->Flask-Limiter<4,>=3.5.0->OctoPrint==1.10.0rc3)
  Using cached mdurl-0.1.2-py3-none-any.whl.metadata (1.6 kB)
Downloading OctoPrint-1.10.0rc3-py2.py3-none-any.whl (3.4 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.4/3.4 MB 8.2 MB/s eta 0:00:00
Using cached pydantic-1.10.12-py3-none-any.whl (158 kB)
Using cached sarge-0.1.7.post1-py2.py3-none-any.whl (18 kB)
Using cached websocket_client-1.6.1-py3-none-any.whl (56 kB)
Using cached argon2_cffi-23.1.0-py3-none-any.whl (15 kB)
Using cached Babel-2.12.1-py3-none-any.whl (10.1 MB)
Using cached blinker-1.6.3-py3-none-any.whl (13 kB)
Using cached cachelib-0.10.2-py3-none-any.whl (18 kB)
Using cached class_doc-0.2.6-py3-none-any.whl (4.7 kB)
Using cached click-8.1.7-py3-none-any.whl (97 kB)
Using cached colorlog-6.8.2-py3-none-any.whl (11 kB)
Using cached emoji-2.11.0-py2.py3-none-any.whl (433 kB)
Using cached feedparser-6.0.11-py3-none-any.whl (81 kB)
Using cached filetype-1.2.0-py2.py3-none-any.whl (19 kB)
Using cached Flask-2.2.5-py3-none-any.whl (101 kB)
Using cached Flask_Assets-2.1.0-py3-none-any.whl (8.5 kB)
Using cached flask_babel-3.1.0-py3-none-any.whl (9.6 kB)
Using cached Flask_Limiter-3.5.1-py3-none-any.whl (28 kB)
Using cached Flask_Login-0.6.3-py3-none-any.whl (17 kB)
Using cached Markdown-3.4.4-py3-none-any.whl (94 kB)
Using cached netaddr-0.8.0-py2.py3-none-any.whl (1.9 MB)
Using cached netifaces2-0.0.22-cp37-abi3-win_amd64.whl (138 kB)
Using cached OctoPrint_FileCheck-2024.3.27-py2.py3-none-any.whl (31 kB)
Using cached OctoPrint_FirmwareCheck-2021.10.11-py2.py3-none-any.whl (33 kB)
Using cached OctoPrint_PiSupport-2023.10.10-py2.py3-none-any.whl (35 kB)
Using cached passlib-1.7.4-py2.py3-none-any.whl (525 kB)
Using cached pathvalidate-2.5.2-py3-none-any.whl (20 kB)
Using cached pkginfo-1.10.0-py3-none-any.whl (30 kB)
Using cached psutil-5.9.8-cp37-abi3-win_amd64.whl (255 kB)
Using cached pylru-1.2.1-py3-none-any.whl (16 kB)
Using cached pyserial-3.5-py2.py3-none-any.whl (90 kB)
Using cached pytz-2024.1-py2.py3-none-any.whl (505 kB)
Using cached PyYAML-6.0.1-cp312-cp312-win_amd64.whl (138 kB)
Using cached requests-2.31.0-py3-none-any.whl (62 kB)
Using cached semantic_version-2.10.0-py2.py3-none-any.whl (15 kB)
Using cached sentry_sdk-1.44.1-py2.py3-none-any.whl (266 kB)
Using cached tornado-6.2-cp37-abi3-win_amd64.whl (425 kB)
Using cached watchdog-2.3.1-py3-none-win_amd64.whl (80 kB)
Using cached Werkzeug-2.2.3-py3-none-any.whl (233 kB)
Using cached wrapt-1.15.0-py3-none-any.whl (22 kB)
Using cached zipstream_ng-1.7.1-py3-none-any.whl (22 kB)
Using cached regex-2023.12.25-cp312-cp312-win_amd64.whl (268 kB)
Using cached setuptools-69.2.0-py3-none-any.whl (821 kB)
Using cached Unidecode-1.3.8-py3-none-any.whl (235 kB)
Using cached wheel-0.43.0-py3-none-any.whl (65 kB)
Using cached certifi-2024.2.2-py3-none-any.whl (163 kB)
Using cached charset_normalizer-3.3.2-cp312-cp312-win_amd64.whl (100 kB)
Using cached idna-3.6-py3-none-any.whl (61 kB)
Using cached ifaddr-0.2.0-py3-none-any.whl (12 kB)
Using cached itsdangerous-2.1.2-py3-none-any.whl (15 kB)
Using cached Jinja2-3.1.3-py3-none-any.whl (133 kB)
Using cached limits-3.10.1-py3-none-any.whl (45 kB)
Using cached MarkupSafe-2.1.5-cp312-cp312-win_amd64.whl (17 kB)
Using cached more_itertools-10.2.0-py3-none-any.whl (57 kB)
Using cached ordered_set-4.1.0-py3-none-any.whl (7.6 kB)
Using cached rich-13.7.1-py3-none-any.whl (240 kB)
Using cached typing_extensions-4.11.0-py3-none-any.whl (34 kB)
Using cached urllib3-2.2.1-py3-none-any.whl (121 kB)
Using cached webassets-2.0-py3-none-any.whl (142 kB)
Using cached argon2_cffi_bindings-21.2.0-cp36-abi3-win_amd64.whl (30 kB)
Using cached colorama-0.4.6-py2.py3-none-any.whl (25 kB)
Using cached cffi-1.16.0-cp312-cp312-win_amd64.whl (181 kB)
Using cached Deprecated-1.2.14-py2.py3-none-any.whl (9.6 kB)
Using cached importlib_resources-6.4.0-py3-none-any.whl (38 kB)
Using cached markdown_it_py-3.0.0-py3-none-any.whl (87 kB)
Using cached packaging-24.0-py3-none-any.whl (53 kB)
Using cached pygments-2.17.2-py3-none-any.whl (1.2 MB)
Using cached mdurl-0.1.2-py3-none-any.whl (10.0 kB)
Using cached pycparser-2.22-py3-none-any.whl (117 kB)
Installing collected packages: webassets, sgmllib3k, sarge, pytz, pyserial, pylru, passlib, OctoPrint-PiSupport, OctoPrint-FirmwareCheck, OctoPrint-FileCheck, netaddr, ifaddr, filetype, zipstream-ng, zeroconf, wrapt, wheel, websocket-client, watchdog, urllib3, unidecode, typing-extensions, tornado, setuptools, semantic-version, regex, PyYAML, pygments, pycparser, psutil, pkginfo, pathvalidate, packaging, ordered-set, netifaces2, more-itertools, mdurl, MarkupSafe, markdown, itsdangerous, importlib-resources, idna, future, frozendict, feedparser, emoji, colorama, charset-normalizer, certifi, cachelib, blinker, Babel, werkzeug, sentry-sdk, requests, pydantic, markdown-it-py, Jinja2, deprecated, colorlog, Click, class-doc, cffi, rich, limits, flask, argon2-cffi-bindings, Flask-Login, Flask-Limiter, Flask-Babel, Flask-Assets, argon2-cffi, OctoPrint
Successfully installed Babel-2.12.1 Click-8.1.7 Flask-Assets-2.1.0 Flask-Babel-3.1.0 Flask-Limiter-3.5.1 Flask-Login-0.6.3 Jinja2-3.1.3 MarkupSafe-2.1.5 OctoPrint-1.10.0rc3 OctoPrint-FileCheck-2024.3.27 OctoPrint-FirmwareCheck-2021.10.11 OctoPrint-PiSupport-2023.10.10 PyYAML-6.0.1 argon2-cffi-23.1.0 argon2-cffi-bindings-21.2.0 blinker-1.6.3 cachelib-0.10.2 certifi-2024.2.2 cffi-1.16.0 charset-normalizer-3.3.2 class-doc-0.2.6 colorama-0.4.6 colorlog-6.8.2 deprecated-1.2.14 emoji-2.11.0 feedparser-6.0.11 filetype-1.2.0 flask-2.2.5 frozendict-2.4.1 future-0.18.3 idna-3.6 ifaddr-0.2.0 importlib-resources-6.4.0 itsdangerous-2.1.2 limits-3.10.1 markdown-3.4.4 markdown-it-py-3.0.0 mdurl-0.1.2 more-itertools-10.2.0 netaddr-0.8.0 netifaces2-0.0.22 ordered-set-4.1.0 packaging-24.0 passlib-1.7.4 pathvalidate-2.5.2 pkginfo-1.10.0 psutil-5.9.8 pycparser-2.22 pydantic-1.10.12 pygments-2.17.2 pylru-1.2.1 pyserial-3.5 pytz-2024.1 regex-2023.12.25 requests-2.31.0 rich-13.7.1 sarge-0.1.7.post1 semantic-version-2.10.0 sentry-sdk-1.44.1 setuptools-69.2.0 sgmllib3k-1.0.0 tornado-6.2 typing-extensions-4.11.0 unidecode-1.3.8 urllib3-2.2.1 watchdog-2.3.1 webassets-2.0 websocket-client-1.6.1 werkzeug-2.2.3 wheel-0.43.0 wrapt-1.15.0 zeroconf-0.132.0 zipstream-ng-1.7.1
```

**Comment by MaleNurse** [comment]
Working now, thank you!
